src/androidTest/java/com/couchbase/lite/DatabaseAttachmentTest.java

```diff
@@ -168,6 +173,28 @@ public void testAttachments() throws Exception {
     }
 
     /**
+     * - (void) test13_GarbageCollectAttachments
+     */
+    public void test13_GarbageCollectAttachments() throws CouchbaseLiteException{
```

Comment:
To be consistent, we shouldn't prefix with test13 or we can prefix all test methods here with testxx.